configureBuild.sh

```diff
@@ -119,7 +119,8 @@ setVariablesForConfigure() {
   local openjdk_core_version=${BUILD_CONFIG[OPENJDK_CORE_VERSION]}
   # test-image target is JDK 11+ only. Set to the empty string
   # as build scripts check whether the variable is non-empty.
-  local openjdk_test_image_path=""
+  local openjdk_test_image_path="test"
```

Comment:
I think this change from the non-empty string will cause issues in later scripts (see the comment above).

Reply:
As I understand this, the only difference here is to enable the test-image for jdk8 like other versions. Likewise for docker-build.sh.

Reply:
OK I updated the comment inline

Reply:
in the java 8 repo there is a dir called test:

https://github.com/AdoptOpenJDK/openjdk-jdk8u/tree/master/test

I think this change will have the effect of packaging and archiving that dir